src/go/install.sh

```diff
@@ -191,15 +191,18 @@ else
 fi
 
 # Install Go tools that are isImportant && !replacedByGopls based on
-# https://github.com/golang/vscode-go/blob/v0.31.1/src/goToolsInformation.ts
+# https://github.com/golang/vscode-go/blob/v0.38.0/src/goToolsInformation.ts
 GO_TOOLS="\
     golang.org/x/tools/gopls@latest \
     honnef.co/go/tools/cmd/staticcheck@latest \
     golang.org/x/lint/golint@latest \
     github.com/mgechev/revive@latest \
-    github.com/uudashr/gopkgs/v2/cmd/gopkgs@latest \
-    github.com/ramya-rao-a/go-outline@latest \
-    github.com/go-delve/delve/cmd/dlv@latest"
+    github.com/go-delve/delve/cmd/dlv@latest \
```

Comment:
Can we add unit tests to validate these tools are installed fine? Thanks! 

Ref: https://github.com/devcontainers/features/blob/main/test/go/test.sh
https://github.com/devcontainers/cli/blob/main/docs/features/test.md

Reply:
@samruddhikhandale 

I've added tests for the tools we included this time. It appears that these tools don't have` ~~ --version` implemented for referencing the version, so I've only written tests for installation paths.

```
// test/go/test.sh
...

# gomodifytags
check "gomodifytags is installed at correct path" bash -c "which gomodifytags | grep /go/bin/gomodifytags"

# goplay
check "goplay is installed at correct path" bash -c "which goplay | grep /go/bin/goplay"

# gotests
check "gotests is installed at correct path" bash -c "which gotests | grep /go/bin/gotests"

# impl
check "impl is installed at correct path" bash -c "which impl | grep /go/bin/impl"

...
```

Reply:
Thank you!